console/module/authorization/src/main/java/org/eclipse/kapua/app/console/module/authorization/client/group/GroupGrid.java

```diff
@@ -86,6 +93,13 @@ protected void load(Object loadConfig,
         columnConfigs.add(columnConfig);
 
         columnConfig = new ColumnConfig("userName", MSGS.gridGroupColumnHeaderCreatedBy(), 200);
+        columnConfig.setRenderer(new GridCellRenderer<GwtGroup>() {
```

Comment:
Uhmmm this column can be namend "CreatedBy" since also the MSGS refers to a `gridGroupColumnHeaderCreatedBy` messages. 

I think that `username` column name is a mistake :) 

After rename, the `CreatedByNameCellRenderer` could be used. Please also remove `getUsername()` and `setUsername()` from GwtGroup  and use `createtBy` instead.